app/src/main/kotlin/at/bitfire/davdroid/ui/setup/GoogleLoginFragment.kt

```diff
@@ -147,13 +148,26 @@ class GoogleLoginFragment(private val defaultEmail: String? = null): Fragment()
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
         val view = ComposeView(requireActivity()).apply {
             setContent {
-                GoogleLogin(defaultEmail = defaultEmail, onLogin = { accountEmail, clientId ->
-                    loginModel.baseURI = googleBaseUri(accountEmail)
-                    loginModel.suggestedAccountName = accountEmail
+                GoogleLogin(defaultEmail = defaultEmail, onLogin = { userEmail, userClientId ->
+                    if (userEmail.isNullOrBlank()) {
```

Comment:
Maybe it's worth thinking about using `StringUtils.trimToNull` (like for the client ID) here too. I usually use it when I expect people to paste things. The clipboard often contains white space before/after the desired content (like `email@adress ` in this case).

Reply:
Probably better safe than sorry. Will add it